packages/credential-sdk/src/types/generic/typed-enum.js

```diff
@@ -87,6 +87,22 @@ class TypedEnum extends withBase(class EnumBase {}) {
       });
 
       const that = this;
+
+      // TODO: revisit this
```

Comment:
todo relevant for this PR or another time?

Reply:
This reminder is for the next time.